krabs/krabs/filtering/event_filter.hpp

```diff
@@ -49,6 +50,14 @@ namespace krabs {
          */
         event_filter(filter_predicate predicate);
 
+        /**
+         * <summary>
+         *   Constructs an event_filter that is setting trace native filters
```

Comment:
nit: `that filters on a specific event id in the provider`.

how do we teach consumers of this API that if one or more provider filters is enabled, no other event IDs will be available to other (fluent) filters?

Reply:
There is no change for functional behavior, filters are joined as with predicate. They are created as list while enabling tracing provider.... for customer is no functional difference just performance improvement